packages/graphql-playground-react/src/components/Playground.tsx

```diff
@@ -69,13 +69,13 @@ export interface Props {
   isEndpoint?: boolean
   isApp?: boolean
   onChangeEndpoint?: (endpoint: string) => void
-  share: (state: any) => void
+  share?: (state: any) => void
```

Comment:
I have made these properties optional because where it's being called (in `PlaygroundWrapper.tsx`) the properties are not bound - so I figured they could not be called.

Not sure what the right fix here is though.

Reply:
whatever avoids changing how user interfaces work. but making an existing property optional is fine, the other way would be breaking